db/seeds.rb

```diff
@@ -1,7 +1,18 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'open-uri'
+
+puts 'CREATING Cards'
+
+Card.delete_all
+
+user_agent = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_7_0) AppleWebKit/535.2 (KHTML, like Gecko) Chrome/15.0.854.0 Safari/535.2"
+
+html = "http://www.languagedaily.com/learn-german/vocabulary/common-german-words"
+
+@doc = Nokogiri::HTML(open(html, 'User-Agent' => user_agent), nil, "UTF-8")
+@doc.css('div.jsn-article-content table tbody tr').each_with_index { |word, index|
+  original_text = word.css('td.bigLetter')
+  translated_text = word.css('td')[2]  
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.